util/helm/client.go

```diff
@@ -414,7 +413,11 @@ func getIndexURL(rawURL string) (string, error) {
 	return repoURL.String(), nil
 }
 
-func (c *nativeHelmChart) GetTags(chart string, noCache bool) (*TagsList, error) {
+type Entries struct {
```

Comment:
Could we simplify this to just be a simple string array instead of a separate type?

Reply:
So, I had a look into it, and this specific structure is used on L496 to unmarshal a JSON response from helm.

We could make it a simple array, and just inline the struct if that would be preferred - would definitely be clearer (and avoid reusing a struct where it shouldn't be)